changelog/allow_casting_from_typetuple_to_typetuple.dd

```diff
@@ -0,0 +1,20 @@
+Casting between compatible tuples
+
+Prior to this release, casting between built-in tuples of the same type was not allowed.
+
+Starting with this release, casting between tuples of the same length is accepted provided that the underlying types of the casted tuple are implicitly convertible to the target tuple types.
+
+---
+alias Tuple(T...) = T;
+
+void foo()
+{
+    Tuple!(int, int) tup;
+
+    auto foo = cast(long) tup;
+    pragma(msg, typeof(foo)); // (int, int)
```

Comment:
@kubo39 Is this correct? `cast(long)` on a sequence of 2 ints seems like it should be an error.

Reply:
Sorry, that doesn't help me. (From your link I found @MoonlightSentinel 
 asking the same question https://github.com/dlang/dmd/pull/13502#discussion_r780600290). 

Why allow casting a sequence of 2 values to a single type?
And why is the result unchanged?